src/hotspot/cpu/riscv/vm_version_riscv.cpp

```diff
@@ -182,6 +182,11 @@ void VM_Version::common_initialize() {
   }
 
   if (UseZicboz) {
+    if (zicboz_block_size.enabled()) {
+      assert(UseZic64b ? zicboz_block_size.value() == 64 : true, "Zicboz block size should be 64 when UseZic64b is true");
+      assert(is_power_of_2(zicboz_block_size.value()), "CacheLineSize must be a power of 2");
```

Comment:
Can we merge the two assertions into one? Like:
`assert(UseZic64b ? zicboz_block_size.value() == 64 : is_power_of_2(zicboz_block_size.value()), "Sanity");`

Reply:
Thanks! Fixed.